lib/Doctrine/MongoDB/EagerCursor.php

```diff
@@ -65,6 +65,7 @@ public function __construct(Cursor $cursor)
     public function count()
     {
         $this->initialize();
+
         return count($this->data);
```

Comment:
If we change the `count()` prototype to `count($foundOnly = false)`, as suggested above, we'll need to add the same here. Unfortunately, a false value for that option doesn't really apply to EagerCursor, since it inherently only counts the found documents after initialization. I'm not sure what the best solution is for that beyond documentation.